packages/client/src/products/__tests__/getProductColorGrouping.test.ts

```diff
@@ -5,43 +5,31 @@ import {
 } from 'tests/__fixtures__/products';
 import client from '../../helpers/client';
 import fixtures from '../__fixtures__/getProductColorGrouping.fixtures';
-import moxios from 'moxios';
+import mswServer from '../../../tests/mswServer';
 
 describe('getProductColorGrouping', () => {
   const expectedConfig = undefined;
   const query = { pageSize: 10 };
   const spy = jest.spyOn(client, 'get');
 
-  beforeEach(() => {
-    moxios.install(client);
-    jest.clearAllMocks();
-  });
-
-  afterEach(() => moxios.uninstall(client));
+  beforeEach(jest.clearAllMocks);
 
   it('should handle a client request successfully', async () => {
     const response = mockProductColorGrouping;
 
-    fixtures.success({
-      id: mockProductId,
-      query,
-      response,
-    });
+    mswServer.use(fixtures.success(response));
 
```

Comment:
This would look great with `expect.assertions(2);` here :)

Reply:
Added 🙂